src/mono/mono/metadata/object.h

```diff
@@ -135,6 +135,9 @@ mono_ldstr		    (MonoDomain *domain, MonoImage *image, uint32_t str_index);
 MONO_API MONO_RT_EXTERNAL_ONLY MonoString*
 mono_string_is_interned	    (MonoString *str);
 
+MONO_API MONO_RT_EXTERNAL_ONLY int
+mono_string_instance_is_interned (MonoString *str_raw);
```

Comment:
Why is this a public API?

Reply:
That doesn't necessarily mean that it needs to be a (stable) public API.

You could put it in mono/metadata/mono-private-unstable.h, for example.
Or in a `-internal.h` header (I believe wasm driver.c has access to those).

Reply:
Sounds good, I'll do that.